packages/wmr/types.d.ts

```diff
@@ -28,7 +38,7 @@ declare module 'wmr' {
 		aliases: Record<string, string>;
 		env: Record<string, string>;
 		middleware: Middleware[];
-		plugins: Plugin[];
+		plugins: WMRPlugin[];
```

Comment:
I have no idea how painful this would be, but I was wondering if we should have `Plugin` be the WMR Plugin interface, and use `RollupPlugin` to refer to base Rollup plugins.